include/sound/sof/header.h

```diff
@@ -76,6 +76,7 @@
 #define SOF_IPC_PM_CLK_REQ			SOF_CMD_TYPE(0x006)
 #define SOF_IPC_PM_CORE_ENABLE			SOF_CMD_TYPE(0x007)
 #define SOF_IPC_PM_GATE				SOF_CMD_TYPE(0x008)
+#define SOF_IPC_PM_VFE_RESET_STATUS		SOF_CMD_TYPE(0x010)
```

Comment:
again I don't understand who does what:

the commit message states that 'the guest will send a topology to the host", while in previous patches the guest retrieves a topology from the host.

??

Reply:
I'll explain in "design"